### DISCUSSION TITLE
docs darkmode

### DISCUSSION THREAD

**Discussion by pizzav-xyz** [zed-industries/zed#14845] [Feature Requests]
how do i turn on darkmode on the docs?

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support

**Comment by pizzav-xyz** [comment]
thanks for this necropost really appreciate it
